### PR TITLE
Adding JWK thumbprint utility method

### PR DESCRIPTION
**Pull request by sgomes**
@crhym3 PTAL! :)

### CODE REVIEW

**Comment by sgomes** [comment]
Rebased on top of master, build now passes.

File: jws.go [hunk 1]
```diff
@@ -56,3 +58,16 @@ type staticNonceSource string
 func (s staticNonceSource) Nonce() (string, error) {
 	return string(s), nil
 }
+
+func jwkThumbprint(key rsa.PublicKey) string {
```
Comment (crhym3) [inline jws.go]: I think all of the empty lines in this func can be removed. It's a simple and short function.

File: jws.go [hunk 2]
```diff
@@ -13,10 +13,12 @@ package goacme
 
 import (
 	"crypto/rsa"
+	"crypto/sha256"
 	"encoding/base64"
 	"encoding/json"
-
+	"fmt"
 	jose "github.com/letsencrypt/go-jose"
```
Comment (crhym3) [inline jws.go]: can you separate `jose` import from everything else. normally, you'd have something like this:

```go
import (
  "stdlib/pkg1"
  "stdlib/pkg2"

  "github.com/xxx"
)
```

In other words, stdlib imports are separated from external packages with an empty line.

File: jws.go (hunk 1 above)
Comment (sgomes) [inline jws.go]: Done.

File: jws.go (hunk 2 above)
Comment (sgomes) [inline jws.go]: Done.

**Comment by crhym3** [comment]
LGTM and submitting. Need this to replace jose pkg.

**Pull request merged**